internal/configs/version1/nginx-plus.tmpl

```diff
@@ -115,6 +115,11 @@ http {
         ssl_certificate /etc/nginx/secrets/default;
         ssl_certificate_key /etc/nginx/secrets/default;
 
+        {{range $setRealIPFrom := $.SetRealIPFrom}}
```

Comment:
here  `$` is used -- `$.SetRealIPFrom`. However, in the same snippet in OSS template you use `.SetRealIPFrom`.

Could you make the two template consistent for the newly added config?  better to use `.SetRealIPFrom` style since that is how it's used throughout the main template

 Same for RealIPHeader and RealIPRecursive.